components/admin_real/mobil/ReservationTable.jsx

```diff
@@ -47,12 +46,17 @@ export default function ReservationTable() {
             });
             context.getTableReservations(initial, end);
           }}
+          sx={{
+            "&.MuiDataGrid-root .MuiDataGrid-cell:focus-within": {
+              outline: "none !important",
+            },
+          }}
```

Comment:
so that rows look like this:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/29632564/201443659-6c2c059a-949c-4b0e-a428-035343c145fd.png">
instead of this:
<img width="317" alt="image" src="https://user-images.githubusercontent.com/29632564/201443676-1397a630-a4b5-4ab9-a995-42117f4ac6b5.png">

Reply:
nice!